libs/genai/pyproject.toml

```diff
@@ -14,6 +14,7 @@ license = "MIT"
 python = ">=3.9,<4.0"
 langchain-core = "^0.3.43"
 google-ai-generativelanguage = "^0.6.16"
+google-genai = "^1.7.0"
```

Comment:
let's avoid bringing an additional dependency here, please

we can use original protobufs and dicts, that would be more than enough for now